fred2/voiceactingmanager.cpp

```diff
@@ -406,7 +406,7 @@ void VoiceActingManager::OnGenerateScript()
 		return;
 
 	CString dlgPathName = dlg.GetPathName( );
-	string_copy(pathname, dlgPathName, 256);
+	string_copy(pathname, dlgPathName, 256 - 1);
```

Comment:
`sizeof()`?

It's especially useful when the declaration of the char buffer is several lines up, as opposed to immediately above.

Reply:
See above